nsist/pyapp.nsi

```diff
@@ -107,22 +107,22 @@ Section "!${PRODUCT_NAME}" sec_app
   [% block install_commands %]
   [% if has_commands %]
     DetailPrint "Setting up command-line launchers..."
-    nsExec::ExecToLog '[[ python ]] -Es "$INSTDIR\_assemble_launchers.py" "$INSTDIR\bin"'
+    nsExec::ExecToLog '"[[ python ]]" -Es "$INSTDIR\_assemble_launchers.py" "[[ python ]]" "$INSTDIR\bin"'
```

Comment:
I've changed it because of the added parameter. When I opened the PR, I didn't notice that it already included quotation marks, so I've change the usage of `[[ python ]]` everywhere else.
But yeah, this might break custom templates. I will change it back and leave the python **parameter** unquoted in the template.